modules/articles/server/controllers/articles.server.controller.js

```diff
@@ -81,7 +81,29 @@ exports.delete = function (req, res) {
  * List of Articles
  */
 exports.list = function (req, res) {
-  Article.find().sort('-created').populate('user', 'displayName').exec(function (err, articles) {
+  // Prepare paging parameters
+  var page = Math.max(0, req.query.page -1); // using a zero-based page index for use with skip()
+  var take = req.query.take || 1000;
```

Comment:
I can changed it to something like 100. I used 1000 just as an example. But you're right, it's too large for a default.